### DISCUSSION TITLE
Sessions and Errors Search Improvements

### DISCUSSION THREAD

**Discussion by mayberryzane** [highlight/highlight#5131] [Ideas]
Hey there 🤠 

This quarter, we're going to be working on some improvements to the sessions / errors search in Highlight. We'd like to get your feedback on any frustrations with the current experience and ideas on how we could make it better.

Possible topics for discussion:
- adding the right filters to a search
- structured (e.g. choosing a filtering key and its value(s)) vs unstructured (e.g. free text) searching
- the usability of the different filtering functions ("is", "contains", "matches")
- filtering by date and using the histogram
- the usefulness of the sessions / errors search versus the sitewide search (cmd + K)
- pagination and search shareability
- finding sessions based on their errors or logs (and vice versa)

Thanks!

**Comment by et** [comment]
## Make searching by error body easier

Searching by the error body would be the most common task a developer would want to do. I just want a free form text box that defaults to a contain search on error body (sort of like how the logs combobox works). Instead, I have to click multiple buttons to even get there (and will likely choose `is` instead of `contains`). Furthermore, the search suggestions just seems stuck. I don't really see a need for search suggestions for this type of field.

![Kapture 2023-04-27 at 08 37 48](https://user-images.githubusercontent.com/58678/234897601-82f3b979-34a0-42a6-8830-9fe548b84ddc.gif)
